packages/bpk-component-button/src/BpkButtonLink.scss

```diff
@@ -18,6 +18,12 @@
 
 @import '~bpk-mixins/index';
 
-.bpk-button--link {
-  @include bpk-button--link;
+.bpk-button {
+  &--link {
+    @include bpk-button--link;
+
+    &--padded {
```

Comment:
This violates BEM, we can't have a modifier on top of a modifier. If you change it to `&-padded` it's ok, because then the end result will be `--link-padded`. 

I know it's silly because it's a single character difference, but gotta be consistent with how we apply the rules!

Reply:
Would it make more sense if it was `bpk-button bpk-button--link bpk-button--padded` as `padded` and `link` are both modifiers?

Reply:
Yeah that sounds perfect!